src/featuremap/ugbio_featuremap/create_somatic_pileup_featuremap.py

```diff
@@ -121,6 +34,7 @@ def merge_vcf_files(tumor_vcf, normal_vcf, out_merged_vcf, n_cpu: int | None = N
     if n_cpu is None:
         n_cpu = os.cpu_count()
     # merging T-N VCF files - this results with records from both tumor and normal VCF files
+    out_merged_full_vcf = out_merged_vcf.replace(".vcf.gz", ".full.vcf.gz")
```

Comment:
assert that out_merged ends with ".vcf.gz" or change to 
```suggestion
    out_merged_full_vcf = out_merged_vcf.replace(".vcf.gz", "") + ".full.vcf.gz"
```